multichain-testing/tools/e2e-tools.js

```diff
@@ -438,6 +449,7 @@ export const makeE2ETools = async (
     setTimeout,
     rpcAddress = 'http://localhost:26657',
     apiAddress = 'http://localhost:1317',
+    retryUntilCondition,
```

Comment:
`setTimeout` is already in the list... I suppose somebody might want the `log` in `retryUntilCondition` to be different from the `log` passed to `makeE2ETools`, but in the common case, we can fill it in for them:

```suggestion
    setTimeout,
    rpcAddress = 'http://localhost:26657',
    apiAddress = 'http://localhost:1317',
    retryUntilCondition = makeRetryUntilCondition({ log, setTimeout }),
```

Reply:
Nice, good thinking